src/yardstick/tool/grype.py

```diff
@@ -246,10 +272,29 @@ def install(
             version,
         ):
             tool_obj = cls._install_from_installer(
-                version=version, path=path, use_cache=use_cache, profile=grype_profile, **kwargs
+                version=version,
+                path=path,
+                use_cache=use_cache,
+                profile=grype_profile,
+                **kwargs,
+            )
+        elif version.startswith("path:"):
```

Comment:
soft preference: there is already a little bit of precedence for local installs in the grype quality test code with `file://` : https://github.com/anchore/vunnel/blob/ee45d4e3e2c69adf609c65ebd94fda21169d41a5/tests/quality/config.yaml#L41

Would you be against changing this from `path:` to `file://` for consistency?

Reply:
I'm taking this comment back -- there is also simple scheme precident more similar to `path:` today (and path is better than file)